src/main/scala/com/databricks/spark/csv/util/TextFile.scala

```diff
@@ -22,7 +22,7 @@ import org.apache.hadoop.mapred.TextInputFormat
 import org.apache.spark.SparkContext
 import org.apache.spark.rdd.RDD
 
-private[csv] object TextFile {
+object TextFile {
```

Comment:
I need it here: https://github.com/quartethealth/spark-fixedwidth/blob/feature/fixed-width-parsing/src/main/scala/com/quartethealth/spark/fixedwidth/package.scala#L18